src/test/java/core/basesyntax/services/impl/ReportCreatorImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.services.ReportCreator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportCreatorImplTest {
+    private ReportCreator reportCreator;
+
+    @BeforeEach
+    void setUp() {
+        reportCreator = new ReportCreatorImpl();
+        Storage.storage.clear();
```

Comment:
I would recomment to move "Storage.storage.clear();" to @AfterEach method in all test classes where you work with storage

Reply:
use @beforeEach to clear storage